starter-code/routes/beach.js

```diff
@@ -0,0 +1,38 @@
+const express        = require("express");
+const beach         = express.Router();
+const Beach          = require("../models/beach");
+
+beach.get('/', (req, res, next) => {
+  Beach.find({}, (err, beach) => {
+   
+    if (err) {return next(err);}
+    res.render('beaches', {beach});
+  });
+});
+
+
+beach.post('/save', (req, res, next) => {
+  console.log('saving');
+
+  let location = {
+    type: 'Point',
+    coordinates: [req.body.longitude, req.body.latitude]
+  };
+
+  const new = new Place({
```

Comment:
Your model is Beach, not Place. Furthermore, you can't call a variable new, try with newBeach.